Resources/public/js/views/services/plugins/ez-objectrelationsloadplugin.js

```diff
@@ -48,6 +48,7 @@ YUI.add('ez-objectrelationsloadplugin', function (Y) {
                 loadingError = false,
                 sourceContent = e.content,
                 destinationContentIds = e.destinationContentIds,
+                withLocation = e.loadLocation || e.loadLocationPath,
```

Comment:
It's better to use `e.hasOwnProperty('loadLocation') || e.hasOwnProperty('loadLocationPath')`

Reply:
I'm checking here if one of those flags is TRUE. See method documentation for more details.

Reply:
Right, good point.